doc/rst/source/coast.rst

```diff
@@ -219,6 +219,10 @@ Optional Arguments
     just certain levels (1-4) then use the full syntax **-W**\ *level*/\ *pen*
     and repeat for each level (pen is not used but required to parse the level correctly).
 
+    Since the resolution of the datasets typically are in the order
+    of 100s of meter, consider reducing the number of decimals by
+    setting :term:`FORMAT_FLOAT_OUT`.
```

Comment:
I found "100s" to be a bit confusing. I first tought in 100 sec-arc. I would prefer "hundreds of meters"

Reply:
Agree - changing.

Reply:
Please check now.